src/targets/targets.test.ts

```diff
@@ -62,7 +62,12 @@ availableTargets()
               const { convert } = new HTTPSnippet(request);
               const result = convert(targetId, clientId); //?
 
-              expect(result).toStrictEqual(expected);
+              // Some `multipart/form-data` tests construct their payload with `\r\n` line breaks,
+              // which can be problematic to save and debug within an editor so let's convert them
+              // to a standard line break instead for the sake of this assertion.
+              expect((result as string).replace(/\r\n/g, '\n').trim()).toStrictEqual(
```

Comment:
would you mind please reverting this: see https://github.com/Kong/httpsnippet/pull/263#discussion_r877632362